Description: Use applications provided by Debian packages.
 In particular:
 * gfanInterface - fig2dev
 * Posets - xdg-open for default pdf viewer
Author: Doug Torrance <dtorrance@piedmont.edu>
Forwarded: not-needed
Last-Update: 2021-01-01

--- a/M2/Macaulay2/packages/gfanInterface.m2
+++ b/M2/Macaulay2/packages/gfanInterface.m2
@@ -15,7 +15,7 @@
 	Keywords => {"Interfaces"},
 	Configuration => {
 		"path" => "",
-		"fig2devpath" => "",
+		"fig2devpath" => "/usr/bin/",
 		"keepfiles" => true,
 		"verbose" => false,
 		"cachePolyhedralOutput" => true
--- a/M2/Macaulay2/packages/Posets.m2
+++ b/M2/Macaulay2/packages/Posets.m2
@@ -21,7 +21,7 @@
         Headline => "partially ordered sets (posets)",
 	Keywords => {"Combinatorics"},
         Configuration => {
-            "DefaultPDFViewer" => "open", -- "open" for Macs and "evince" for Linux
+            "DefaultPDFViewer" => "xdg-open", -- "open" for Macs and "evince" for Linux
             "DefaultPrecompute" => true,
             "DefaultSuppressLabels" => true
             },
